maintenance/README.md

```diff
@@ -4,58 +4,56 @@
 
 The Maintenance Reference Ontology supports modelling of concepts associated with asset reliability and maintenance management. It is relevant for data captured at the design stage of the asset life cycle (e.g. in failure modes and effects analysis) and in the operations and maintenance phase of the life cycle (e.g. asset reliability and the maintenance activities associated with delivering value from use of the assets).
 
-This Maintenance Reference Ontology is a minimal ontology aligned to IOF CORE and by extension BFO. It deliberately captures only concepts frequently captured in data in the maintenance phase of the asset life cycle. 
+This Maintenance Reference Ontology is a minimal ontology aligned to IOF CORE and by extension BFO. It deliberately captures only concepts frequently captured in data in the maintenance phase of the asset life cycle.
 
-This minimal ontology supports the use of modular ontologies for specific classes of interest and application ontologies (examples below). Proposed modular ontologies include asset functions, FMEA codes, maintenance activities. Each of these can be attached to this Reference Ontology. 
+This minimal ontology supports the use of modular ontologies for specific classes of interest and application ontologies (examples below). Proposed modular ontologies include asset functions, FMEA codes, maintenance activities. Each of these can be attached to this Reference Ontology.
 
 This Maintenance Reference Ontology includes the following classes and object properties. Each defined class has natural language, first order logic and a semi-formal definition. Each primitive has a natural language definition and necessary or sufficient axioms as appropriate.
 
 ![Taxonomy and relationships of the Maintenance Reference Ontology](images/MaintenanceDevRelationships.png)
 
+### Glossary of terms
+
+A glossary of entities and relations in this ontology is at https://uwasystemhealth.github.io/IOF-MRO-Glossary/
```

Comment:
Should this  glossary be made available as a public wiki page on the IOF/OAGi confluence and be referenced instead?